scripts/zones/Southern_San_dOria/npcs/Balasiel.lua

```diff
@@ -31,13 +31,13 @@ end
 function onTrigger(player,npc)
     local wsQuestEvent = dsp.wsquest.getTriggerEvent(wsQuest,player)
     local lvl = player:getMainLvl()
-    local aSquiresTest = player:getQuestStatus(SANDORIA, A_SQUIRE_S_TEST)
-    local aSquiresTestII = player:getQuestStatus(SANDORIA,A_SQUIRE_S_TEST_II)
-    local aKnightsTest = player:getQuestStatus(SANDORIA, A_KNIGHT_S_TEST)
+    local aSquiresTest = player:getQuestStatus(SANDORIA, dsp.quest.id.sandoria.A_SQUIRE_S_TEST)
+    local aSquiresTestII = player:getQuestStatus(SANDORIA,dsp.quest.id.sandoria.dsp.quest.id.sandoria.A_SQUIRE_S_TEST_II)
```

Comment:
double match

Reply:
Indeed. I was an idiot at the start for the San d'Oria replace and was matching for the var names instead of the quest functions themselves (like in all commits afterward). Thought I got all these, but apparently not. Thanks wren; is fixed for this next commit I'm going through.